win-spout.cpp

```diff
@@ -139,36 +145,78 @@ static bool win_spout_sender_has_changed(win_spout * context)
 static void win_spout_init(void *data, bool forced = false)
 {
 	struct win_spout *context = (win_spout *)data;
-	if (context->initialized)
+	if (context->initialized) {
+		context->spout_status = 0;
```

Comment:
Thank you for adding these error codes (`spout_status` and `render_status`).

Where do you plan to access them? It's nice having numerical codes if you're going to use them somewhere, but I can't see where these are read, so maybe it's cleaner without them?

Reply:
I have never think of use them as error codes .I just use them to avoid log storm while same error keep happen.

Reply:
Ah yes. OK that is a good idea. Thanks for explaining.